MonstieWash/Assets/Scripts/UIManager.cs

```diff
@@ -36,19 +50,105 @@ private void ToggleUIVisibility()
         CBAnimator.SetBool("Hide", m_UIVisible);
     }
 
-    // Update is called once per frame
-    private void Update()
+/// <summary>
+/// Public method used by TaskTracker to initialise clipboard after all task trackers have been initialised. Creates filters for tasks to show on each scene.
+/// </summary>
+/// <param name="keys">A list of all the keys used to identify tasks tracked in TaskTracker</param>
+    public void LoadKeys(List<string> keys)
+    {
+        m_TaskKeys = keys;
+        foreach (var scene in m_roomSaver.m_allScenes)
+        {
+            switch (scene)
+            {
+                case "Overview":
+                    m_SceneTasks.Add(scene, m_TaskKeys.FindAll(s => s.Contains("Overall") && s.Count(x => x == '#') <= 1));
+                    break;
+                default:
+                    var sceneKeys = m_TaskKeys.FindAll(s => s.Contains(scene));
+                    sceneKeys.Add("Overall");
+                    m_SceneTasks.Add(scene, sceneKeys);
+                    break;
+            } 
```

Comment:
I might be missing something, but I'm not sure why this isn't just an if-else since there's only two options? Feel free to leave it if you have plans for it to be expanded on later on but if this is how it'll stay it might make it easier to read turning it into a simple `if (scene == "Overview") ... else ...`

Reply:
This makes sense, I was considering the possibility for other cases when I originally wrote it but it might not be necessary with the current design.